frontend/package.json

```diff
@@ -133,7 +134,7 @@
     "geostyler-geocss-parser": "https://github.com/allyoucanmap/geostyler-geocss-parser/tarball/build",
     "geostyler-openlayers-parser": "1.1.4",
     "geostyler-sld-parser": "2.0.1",
-    "history": "4.6.1",
+    "history": "^4.7.2",
```

Comment:
We always use precise versions for packages. Can you change this to "4.7.2"?
Please, also do the same for other similar packages.